font-awesome/src/main/scala/lucuma/react/fa/interop.scala

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.react.fa
+
+import cats.syntax.all.*
+import japgolly.scalajs.react.*
+import japgolly.scalajs.react.vdom.*
+
+import scalajs.js
+
+extension (ae: AbstractElement)
```

Comment:
I guess this is based on what the js library does?

Reply:
Yes, the JS library returns an "abstract" representation of the tag + attributes + children and the react library turns that into Vdom. Here we do it with scalajs-react idioms.